repobuilder/job.go

```diff
@@ -143,6 +140,20 @@ func NewRepoBuilderJob(opts JobOptions) (amboy.Job, error) {
 		}
 	}
 
+	repoName := j.getPackageLocation()
+	scopes := []string{}
+	for _, repo := range opts.Distro.Repos {
+		switch opts.Distro.Type {
+		case RPM:
+			scopes = append(scopes, path.Join(repo, repoName, opts.Arch))
+		case DEB:
+			scopes = append(scopes, path.Join(repo, repoName))
```

Comment:
no `opts.Arch`?

Reply:
nope, the scope of the lock for debian distros has to be broader because the Release and Release.gpg are updated by all repositories

http://repo.mongodb.com/apt/ubuntu/dists/xenial/mongodb-enterprise/4.2/